src/errors/InterfaceInstantiationError.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * AbstractInstantiationError is thrown when an interface is instantiated directly.
```

Comment:
Copypasta: `AbstractInstantiationError` isn't the class this comment is describing.